### PR TITLE
add license and setup files

### PR DESCRIPTION
**Pull request by mqharris**
I think this is all we need for this task.

Pushed it to PyPi

you can try it your self with:
`pip install quickscreen`

I think the import path is probably a little janky but I was able to access my Plotter() object with
`from quickscreen.plot.plotter import Plotter`
then call it with
`p = Plotter(None)`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+setup(
+    name='quickscreen',
+    version='0.1',
+    packages=find_packages(exclude=['tests*']),
+    license='MIT',
+    description='A package for exploring data',
+    url='https://github.com/mqharris/533_lab2',
+    author='Nathan Smith, Mitch Harris, Ryan Koenig',
+    author_email='blank'
```
Comment (mqharris) [inline setup.py:11]: Left this blank
But I might need to put an email in that is registered to PyPi

Comment (WraySmith) [reply setup.py:11]: how is the description.md, demo.ipynb, and data folder getting screened out?

File: setup.py [hunk 2]
```diff
@@ -3,7 +3,12 @@
 setup(
     name='quickscreen',
     version='0.1',
-    packages=find_packages(exclude=['tests*']),
+    packages=find_packages(exclude=[
+        'tests*',
+         "data", 
+         "example.ipynb",
+         "description.md"
+         ]),
```
Comment (WraySmith) [inline setup.py:11]: and I guess the README as well as it's github repo specific?

Comment (mqharris) [reply setup.py:11]: I'm looking at this right now, I may not need to list anything that isn't a directory...

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+setup(
+    name='quickscreen',
+    version='0.2',
+    packages=["quickscreen"],
+    install_requires=["pandas", "numpy", "matplotlib", "seaborn", "scikit-learn"],
+    license='MIT',
+    description='A package for exploring data',
+    url='https://github.com/mqharris/533_lab2',
+    author='Nathan Smith, Mitch Harris, Ryan Koenig',
+    author_email='mitchharris@live.ca'
```
Comment (mqharris) [inline setup.py:12]: it needed an email for publishing

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='quickscreen',
+    version='0.2',
+    packages=["quickscreen"],
```
Comment (mqharris) [inline setup.py:6]: this appears to have worked, and is simpler

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='quickscreen',
+    version='0.2',
```
Comment (mqharris) [inline setup.py:5]: :tada:

**Comment by WraySmith** [comment]
I guess the path can be remedied by adding the shortcuts in the __init__.py files in each directory folder as noted in the lecture - no real need though

**Review by RyKoe — APPROVED** [review]
looks good

**Pull request merged**